.cspell.json

```diff
@@ -241,6 +241,7 @@
     "authcli",
     "autoclean",
     "Autocompute",
+    "Autocore",
```

Comment:
I think it's `AutoCore`, and there is no need to add it to the dictionary because it's a combination of `Auto` and `Core`.

```suggestion
```

Reply:
https://autocore.ai/

Reply:
If you want to add this, `autocore` is better.

Reply:
I updated my suggestion in that sensing calibration doc PR to change `Autocore` to `AutoCore`, so if that fixes the spellcheck issue then I'm happy to remove the word.